mula/scheduler/schedulers/boefje.py

```diff
@@ -457,8 +457,8 @@ def is_task_rate_limited(self, task: BoefjeTask, hit: bool = True) -> bool:
         with self.rate_limiter_lock:
             # Get the identifier for the rate limiter
             identifier_template = rate_limit.identifier
-            environment = Environment()
-            identifier = environment.get_template(identifier_template).render(task=task)
+            environment = Environment(loader=BaseLoader())
+            identifier = environment.from_string(identifier_template).render(task=task)
```

Comment:
@ammar92 not yet sure if this is the way, but implemented this is the tests. So change it if necessary